File: src/main/java/com/qfedu/entity/Goods.java
package com.qfedu.entity;

import javax.persistence.*;

public class Goods {

    private int id;
    private String name;
    private String code;
    private String color;
    private String size;

    private double price;
    private int amount;

    private int deleteFlag;

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }

    public int getDeleteFlag() {
        return deleteFlag;
    }

    public void setDeleteFlag(int deleteFlag) {
        this.deleteFlag = deleteFlag;
    }


}

File: src/main/java/com/qfedu/entity/Goods.kt
package com.qfedu.entity

import java.math.BigDecimal
import java.util.Date
import javax.persistence.Column
import javax.persistence.Id
import javax.persistence.Table

@Table(name = "t_goods")
class Goods {
    @Id
    var id: Int? = null
    @Column(name = "goods_name")
    var goodsName: String? = null
    @Column(name = "price")
    var price: BigDecimal? = null
    @Column(name = "stock")
    var stock: Long? = null

    @Column(name = "operator_id")
    var operatorId: Int? = null
    @Column(name = "operator_name")
    var operatorName: String? = null
    @Column(name = "operator_time")
    var operatorTime: Date? = null

}
